src/main/groovy/com/ullink/NuGetExtension.groovy

```diff
@@ -2,4 +2,5 @@ package com.ullink
 
 class NuGetExtension {
     String version = '3.4.3'
+    String nugetUrl = null
```

Comment:
How would you reccommend handling the 'latest' version then?  Should using the latest version only work if they specify the entire url?